<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrescriptionEditView</class>
 <widget class="QWidget" name="PrescriptionEditView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>处方编辑</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>处方信息</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>处方号：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtPrescriptionNumber">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>系统自动生成</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>开方日期：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDateTimeEdit" name="datePrescriptionDate">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>患者：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="txtPatientId">
          <property name="placeholderText">
           <string>请输入患者ID或姓名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSelectPatient">
          <property name="text">
           <string>选择患者</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>医生：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="txtDoctorId">
          <property name="placeholderText">
           <string>请输入医生ID或姓名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSelectDoctor">
          <property name="text">
           <string>选择医生</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>诊断：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="3">
       <widget class="QLineEdit" name="txtDiagnosis">
        <property name="placeholderText">
         <string>请输入诊断信息</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>备注：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="3">
       <widget class="QPlainTextEdit" name="txtNotes">
        <property name="maximumHeight">
         <number>60</number>
        </property>
        <property name="placeholderText">
         <string>请输入处方备注信息...</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>支付状态：</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="cmbPaymentStatus"/>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>发药状态：</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QComboBox" name="cmbDispensingStatus"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>处方总金额：</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="lblTotalAmount">
        <property name="text">
         <string>0.00 元</string>
        </property>
        <property name="styleSheet">
         <string notr="true">font-weight: bold; color: #e74c3c; font-size: 14px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>处方药品详情</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="btnAddMedicine">
          <property name="text">
           <string>添加药品</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDeleteMedicine">
          <property name="text">
           <string>删除药品</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lblMedicineCount">
          <property name="text">
           <string>药品数量：0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblMedicineTotal">
          <property name="text">
           <string>药品总金额：0.00 元</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tableViewDetails">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblError">
     <property name="text">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">color: red;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>